src/styles/card.scss

```
@@ -0,0 +1,116 @@
+.product-card {
+  grid-auto-flow: row;
+  padding: 50px 40px;
+  align-items: center;
+  justify-content: center;
+  display: grid;
+  grid-template-columns: repeat(1, $card-size);
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-size);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-size);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-size);
+  }
+}
+
+.card {
+  padding: 0 16px;
+  flex-direction: column;
+  display: grid;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $white;
+  width: 200px;
+
+  &:hover {
+    color: #34568b;
+    transform: scale(1.2);
+  }
+
+
+  &__img {
+    margin: 32px 0 40px 3px;
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+  }
+
+  &__title {
+    margin: 0;
+    width: 166px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    margin-bottom: 0;
+    color: $secondary;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    text-align: center;
+    margin-top: 24px;
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .price {
+  justify-content: center;
+  display: flex;
+  color: $secondary;
+  font-size: 12px;
+  line-height: 18px;
+  }
+
+  &__price-text {
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    color: $main-accent;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    margin: 16px 0;
+    color: $white;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    border-radius: 5px;
+    background: $blue-accent;
+    width: 166px;
+    height: 40px;
+    flex-shrink: 0;
+    border: none;
+  }
+
+  .card__button:hover {
```

why don't u nest it like the above selectors into .card styles like
```suggestion
  &__button {
          &:hover {
```
?